### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by chrislovecnm**
- updated the section about ip masquerading since it was inaccurate since #187 
- made some other notes about the private and beta modules.

Have a comment in a section, because I don't quite understand what we are trying to communicate to the user.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ The resources/services/activations/deletions that this module will create/trigge
 - Create GKE Node Pool(s) with provided configuration and attach to cluster
 - Replace the default kube-dns configmap if `stub_domains` are provided
 - Activate network policy if `network_policy` is true
-- Add `ip-masq-agent` configmap with provided `non_masquerade_cidrs` if `network_policy` is true
+- Add `ip-masq-agent` configmap with provided `non_masquerade_cidrs` if `configure_ip_masq` is true
+
+Sub modules are provided from creating private clusters, beta private clusters, and beta public clusters as well.  Beta sub modules allow for the use of various GKE beta features. See the modules directory for the various sub modules.
```
Comment (morgante) [inline README.md]:
```suggestion
Su modules are provided from creating private clusters, beta private clusters, and beta public clusters as well.  Beta sub modules allow for the use of various GKE beta features. See the [modules directory](./modules) for the various submodules.
```

File: README.md [hunk 2]
```diff
@@ -90,6 +92,7 @@ module "gke" {
 }
 ```
 
+<!-- do not understand what this is about -->
```
Comment (morgante) [inline README.md]: These are just the steps to run Terraform. In theory you could copy & paste the above config into a Terraform file then follow these steps to deploy the GKE cluster.

File: README.md (hunk 1 above)
Comment (morgante) [reply README.md]: We should probably exclude this block in submodules.

File: README.md (hunk 2 above)
Comment (chrislovecnm) [reply README.md]: I will add what you wrote and it will make more sense to me.

File: README.md (hunk 1 above)
Comment (aaron-lane) [reply README.md]: I think this was intended to be `Submodules` rather than `Su modules`.

**Pull request merged**